content/article/what-is-new-in-perl.md

```diff
@@ -79,8 +79,8 @@ class Employee {
     field $name :param;
     field $age  :param;
 
-    method name { return $name }
-    method age  { return $age  }
+    method name    { return $name }
+    method get_age { return $age  }
```

Comment:
This is not needed, instead line 88 needs updating.

Reply:
disagree: lines 87-88 should be the same as 111-112
the reason to show how class could be simplified with the same code

Reply:
> disagree: lines 87-88 should be the same as 111-112 the reason to show how class could be simplified with the same code

interesting point, my initial thought was keep the getter simple in the first example then show the same with reader. since reader has two different variants, so I came up with the two different approaches.